ci/test_conda_python.sh

```diff
@@ -9,26 +9,31 @@ set -euo pipefail
 rapids-logger "Starting Conda Package Test"
 
 rapids-logger "Creating Test Environment"
-rapids-mamba-retry create -n test
+# TODO: replace this with rapids-dependency-manager
+rapids-mamba-retry create -n test \
+  -c `pwd`/conda-repo \
+  click \
+  pytest \
+  clangdev >=18 \
+  cuda-nvcc >==${RAPIDS_CUDA_VERSION%.*} \
+  cuda-version >==${RAPIDS_CUDA_VERSION%.*} \
```

Comment:
Greater? This should pin to the exact CUDA version defined in the CI image. Also `cuda-nvcc` is constrained by `cuda-version` so we don't need to pin it. Generally we only pin `cuda-version` and not CUDA packages.
```suggestion
  cuda-nvcc \
  cuda-version =${RAPIDS_CUDA_VERSION%.*} \
```